src/pyFAI/gui/widgets/WorkerConfigurator.py

```diff
@@ -550,15 +578,9 @@ def __savePoni(self):
             return
 
         filename = dialog.selectedFiles()[0]
-        config = self.getConfig()
-        if config.get("wavelength") is None:
-            config.pop("wavelength")
-        from ...detectors import detector_factory
-        detector = detector_factory(config.get("detector"), config.get("detector_config"))
-        from ...geometry import Geometry
-        ai = Geometry(detector=detector)
-        ai.set_config(config)
-        ai.save(filename)
```

Comment:
Here, I'm not sure why saving the .poni file has to include all these lines (to be discussed)